cypress/integration/inventory/Item-status-date-updates.spec.js

```diff
@@ -0,0 +1,440 @@
+import uuid from 'uuid';
+import getRandomPostfix from '../../support/utils/stringTools';
+import TestTypes from '../../support/dictionary/testTypes';
+import Orders from '../../support/fragments/orders/orders';
+import NewOrder from '../../support/fragments/orders/newOrder';
+import TopMenu from '../../support/fragments/topMenu';
+import Helper from '../../support/fragments/finance/financeHelper';
+import OrdersHelper from '../../support/fragments/orders/ordersHelper';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import ItemVeiw from '../../support/fragments/inventory/inventoryItem/itemVeiw';
+import Receiving from '../../support/fragments/receiving/receiving';
+import permissions from '../../support/dictionary/permissions';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+import SwitchServicePoint from '../../support/fragments/service_point/switchServicePoint';
+import NewRequest from '../../support/fragments/requests/newRequest';
+import CheckOut from '../../support/fragments/check-out/checkOut';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints';
+import NewServicePoint from '../../support/fragments/service_point/newServicePoint';
+import NewLocation from '../../support/fragments/settings/tenant/locations/newLocation';
+import Locations from '../../support/fragments/settings/tenant/locations/locations';
+import UsersSearchPane from '../../support/fragments/users/usersSearchPane';
+import UsersCard from '../../support/fragments/users/usersCard';
+import ConfirmItemInModal from '../../support/fragments/check-in-actions/confirmItemInModal';
+import UserLoans from '../../support/fragments/users/loans/userLoans';
+import ConfirmItemStatusModal from '../../support/fragments/users/loans/confirmItemStatusModal';
+import RenewConfirmationModal from '../../support/fragments/users/loans/renewConfirmationModal';
+import OverrideAndRenewModal from '../../support/fragments/users/loans/overrideAndRenewModal';
+import ConfirmMultiplePiecesItemCheckOut from '../../support/fragments/check-out/confirmMultiplePiecesItemCheckOut';
+import {
+  REQUEST_POLICY_NAMES,
+  NOTICE_POLICY_NAMES,
+  OVERDUE_FINE_POLICY_NAMES,
+  CY_ENV,
+  LOAN_POLICY_NAMES,
+  LOST_ITEM_FEES_POLICY_NAMES,
+} from '../../support/constants';
+import Requests from '../../support/fragments/requests/requests';
+
+describe('ui-inventory: Item status date updates', () => {
+  const order = { ...NewOrder.defaultOrder };
+  const instanceTitle = `autotest_title_${getRandomPostfix()}`;
+  const orderLine = {
```

Comment:
Move this long object into fragment please
Use here somethig like

> const order = { ...NewOrder.defaultOrder };

Reply:
Done